packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -25,6 +25,7 @@ export interface ILocalFileDestinationProviderOptions {
   file: {
     path: string;
     maxSize?: number;
+    maxSizeJsonl?: number;
```

Comment:
I thought about `maxSizeInternal` (which would be like 'individual') but we shouldn't be splitting media files, so that's not right. I couldn't come up with a good option, which is why I went with jsonl. I's fine to 'leak' it because the only point of allowing this to be split is if the user wants to keep them smaller to making viewing them easier. If we want the backup file to be a black box, we shouldn't provide this option at all.

Reply:
Make sense, thanks for the thought